blazor/dropdown-tree/filtering.md

```diff
@@ -133,4 +133,64 @@ The Data items can be filtered with or without case sensitivity using the [Ignor
 
 You can use [FilterBarPlaceholder](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfDropDownTree-2.html#Syncfusion_Blazor_Navigations_SfDropDownTree_2_FilterBarPlaceholder) to accept the value to be displayed as a watermark text on the filter bar TextBox. `FilterBarPlaceholder` is applicable when `AllowFiltering` is used as true. `FilterBarPlaceholder` is depends on `AllowFiltering` property.
 
-![Blazor Dropdown Tree with local data filtering](./images/filter/blazor-dropdowntree-placeholder.png)
\ No newline at end of file
+![Blazor Dropdown Tree with local data filtering](./images/filter/blazor-dropdowntree-placeholder.png)
+
+## Minimum filter length
+
+When filtering the tree items, you can set the minimum filtering length. This can be done by manual validation by using the [Filtering event arguments](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.DdtFilteringEventArgs.html#Syncfusion_Blazor_Navigations_DdtFilteringEventArgs_Text) within the [Filtering](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfDropDownTree-2.html#Syncfusion_Blazor_Navigations_SfDropDownTree_2_Filtering) event handler.
```

Comment:
When filtering the tree items, you can set the minimum characters after which the filtering must occur. This can be done by checking the text length using the [Filtering event arguments]